packages/governance/src/types.js

```diff
@@ -404,14 +405,18 @@
  */
 
 /**
- * @callback GetParams - getParams() retrieves a Record containing
- *   keyword pairs with descriptions of parameters under governance.
- * @returns {Record<Keyword,ParamShortDescription>}
+ * @callback GetGovernedVaultParams
+ * @returns {{
+ *  'InitialMargin': ParamRecord<'ratio'> & { value: Ratio },
+ *  'InterestRate': ParamRecord<'ratio'> & { value: Ratio },
+ *  'LiquidationMargin': ParamRecord<'ratio'> & { value: Ratio },
+ *  'LoanFee': ParamRecord<'ratio'> & { value: Ratio },
+ * }}
  */
 
 /**
  * @typedef {Object} ParamManagerBase
- * @property {GetParams} getParams
+ * @property {() => Record<Keyword,ParamShortDescription>} getParams
```

Comment:
```suggestion
 * @property {() => Record<Keyword, ParamShortDescription>} getParams
```

Reply:
Thanks. Among the downside of type defs being in jsdoc comments.